platforms/documentation/docs/src/docs/userguide/dep-man/02-declaring-dependency-versions/single_versions.adoc

```diff
@@ -15,28 +15,62 @@
 [[single-version-declarations]]
 = Declaring Versions and Ranges
 
+You can declare dependencies with specific versions or version ranges to define the acceptable versions of a dependency that your project can use:
+
+[source,groovy]
+----
+dependencies {
+    implementation 'org.springframework:spring-core:5.3.8'
+    implementation 'org.springframework:spring-core:5.3.+'
+    implementation 'org.springframework:spring-core:latest.release'
+    implementation 'org.springframework:spring-core:[5.2.0, 5.3.8]'
+    implementation 'org.springframework:spring-core:[5.2.0,)'
+}
+----
+
+== Understanding version declaration
+
 The simplest version declaration is a _simple string_ representing the version to use.
 Gradle supports different ways of declaring a version string:
 
-* An exact version: e.g. `1.3`, `1.3.0-beta3`, `1.0-20150201.131010-1`
-* A Maven-style version range: e.g. `[1.0,)`, `[1.1, 2.0)`, `(1.2, 1.5]`
-** The `[` and `]` symbols indicate an inclusive bound; `(` and `)` indicate an exclusive bound.
-** When the upper or lower bound is missing, the range has no upper or lower bound.
-** The symbol `]` can be used instead of `(` for an exclusive lower bound, and `[` instead of `)` for exclusive upper bound. e.g `]1.0, 2.0[`
-** An upper bound exclude acts as a prefix exclude.
-This means that `[1.0, 2.0[` will also exclude all versions starting with `2.0` that are smaller than `2.0`.
-For example versions like `2.0-dev1` or `2.0-SNAPSHOT` are no longer included in the range.
-* A _prefix_ version range: e.g. `1.\+`, `1.3.+`
-** Only versions exactly matching the portion before the `+` are included.
-** The range `+` on its own will include any version.
-* A `latest-status` version: e.g. `latest.integration`, `latest.release`
-** Will match the highest versioned module with the specified status. See link:{javadocPath}/org/gradle/api/artifacts/ComponentMetadata.html#getStatus--[ComponentMetadata.getStatus()].
-* A Maven `SNAPSHOT` version identifier: e.g. `1.0-SNAPSHOT`, `1.4.9-beta1-SNAPSHOT`
+[cols="1,1,~"]
+|===
+|Version |Example |Note
```

Comment:
Looking at the generated table, this `Version` column looks very squished. Is there a way to make it a bit wider?